odl/tomo/geometry/conebeam.py

```diff
@@ -114,17 +119,31 @@ def __init__(self, apart, dpart, src_radius, det_radius,
         >>> geom.det_axis_init
         array([ 1.,  0.])
 
+        Specifying curvature of the detector:
+
+        >>> apart = odl.uniform_partition(0, 2 * np.pi, 10)
+        >>> dpart = odl.uniform_partition(-np.pi / 2, np.pi / 2, 10)
+        >>> geom = FanBeamGeometry(apart, dpart, src_radius=1, det_radius=5,
+        ...                        det_curve_radius=10)
+        >>> geom.src_position(0)
+        array([ 0., -1.])
+        >>> geom.det_refpoint(0)
+        array([ 0.,  5.])
+        >>> np.round(geom.det_point_position(0, np.pi / 6), 2)
+        # (0, 5) + 10 * (sin(pi/6), cos(pi/6) - 1)
+        array([ 5.  ,  3.66])
```

Comment:
Does this test pass? The comment on the first output line may make it fail. If so, you need to put it on an input line, i.e., write `>>> # Your comment`

Reply:
On my machine, this test indeed fails:
```
132         >>> np.round(geom.det_point_position(0, np.pi / 6), 2)
Expected:
    # (0, 5) + 10 * (sin(pi/6), cos(pi/6) - 1)
    array([ 5.  ,  3.66])
Got:
    array([ 5.  ,  3.66])
```
Together with 2 others:
```
/home/hkohr/git/odl/odl/tomo/geometry/conebeam.py:132: DocTestFailure
________________________ [doctest] odl.tomo.geometry.detector.CircularDetector.surface ________________________
730             shape ``(2,)``, otherwise ``param.shape + (2,)``.
731 
732         Examples
733         --------
734         The method works with a single parameter, resulting in a single
735         vector:
736 
737         >>> part = odl.uniform_partition(-np.pi / 2, np.pi / 2, 10)
738         >>> det = CircularDetector(part, axis=[0, 1], radius=2)
739         >>> np.allclose(det.surface(-np.pi / 2), [-2, -2])
Expected:
    True
Got:
    False

/home/hkohr/git/odl/odl/tomo/geometry/detector.py:739: DocTestFailure
_____________________ [doctest] odl.tomo.geometry.detector.CircularDetector.surface_deriv _____________________
792         --------
793         surface
794 
795         Examples
796         --------
797         The method works with a single parameter, resulting in a single
798         vector:
799 
800         >>> part = odl.uniform_partition(-np.pi / 2, np.pi / 2, 10)
801         >>> det = CircularDetector(part, radius=2)
UNEXPECTED EXCEPTION: TypeError("__init__() missing 1 required positional argument: 'axis'",)
Traceback (most recent call last):

  File "/home/hkohr/miniconda/envs/odl_py3/lib/python3.6/doctest.py", line 1330, in __run
    compileflags, 1), test.globs)

  File "<doctest odl.tomo.geometry.detector.CircularDetector.surface_deriv[1]>", line 1, in <module>

TypeError: __init__() missing 1 required positional argument: 'axis'

/home/hkohr/git/odl/odl/tomo/geometry/detector.py:801: UnexpectedException
```

Reply:
How do you run the tests?

Reply:
Ok, I guess its doctests...

Reply:
Yes, I cloned your branch and ran the tests. If you have `pytest` installed you can run the doctests in a specific module or a directory by
```
pytest odl/tomo/geometry/detector.py  # runs doctests in that module
pytest odl/tomo/geometry/  # runs doctests in all modules in that subdirectory
```

Reply:
Oh, I see, thanks. It should be fixed now